tpcx_bb/benchmark_runner/selene/rapids-tpcx-bb-cuda11.yml

```diff
@@ -0,0 +1,33 @@
+channels:
+  - blazingsql-nightly/label/cuda11.0
+  - rapidsai-nightly
+  - nvidia
+  - conda-forge
+  - defaults
+
+dependencies:
+  - python=3.8
+  - cudatoolkit=11.0
+  - cudf
+  - rmm
+  - dask-cuda
+  - dask-cudf
+  - cuml
+#   - ucx-py
+#   - ucx-proc=*=gpu
+  - blazingsql
+  - scipy
+  - scikit-learn
+  - spacy
+  - oauth2client
+  - asyncssh
+  - psutil
+  - ipykernel
+  - jupyterlab
+  - gspread
+  - oauth2client
+  - pip
+  - pip:
+    - jupyter-server-proxy
+    - git+https://github.com/dask/dask.git
+    - git+https://github.com/dask/distributed.git
```

Comment:
May not be relevant in general but may want to pin these to a specific commit if needed

Reply:
For now, going to leave unpinned as we're keeping a history of the containers.